omeroweb/settings.py

```diff
@@ -53,15 +53,16 @@
 # A boolean that turns on/off debug mode.
 # handler404 and handler500 works only when False
 if 'OMERO_HOME' in os.environ:
-    OMERO_HOME = os.environ.get('OMERO_HOME')
-elif 'OMERODIR' in os.environ:
-    OMERO_HOME = os.environ.get('OMERODIR')
+    logger.warn("OMERO_HOME usage is deprecated in OMERO.web")
```

Comment:
```suggestion
    logger.warn("OMERO_HOME usage is ignored in OMERO.web")
```
Same comment as https://github.com/ome/omero-py/pull/148/files#r358707608